docs/tutorials/auth.md

```diff
@@ -223,3 +223,25 @@ for which it needs a secret key. This key is in a file which can be rotated peri
 because it is reread by Skipper. The path to this file can be passed with the flag
 `-oidc-secret-file` when Skipper is started.
 
+### Authorization and access control
+
+Authorization validation and access control is available by means of a subsequent filter `oidcClaimsQuery`. It inspects the ID token, that is present after a successful `oauthOidc*` filter step, and validates the defined query with the request path.
```

Comment:
Hi there @universam1,

should we include a link to the reference here? Might be easier to deep-dive then.

Reply:
sure, added a link